kafka-client/src/main/java/dev/responsive/kafka/internal/db/rs3/PssPartitioner.java

```diff
@@ -0,0 +1,9 @@
+package dev.responsive.kafka.internal.db.rs3;
+
+import java.util.List;
+
+public interface PssPartitioner {
```

Comment:
Yeah that's fair. I can write something up to go along with a follow-up PR. Since we're all here, a quick breakdown:
LS - stands for logical store. This is 1:1 with a Kafka Streams store at the processor level - meaning it encompasses the entire keyspace (not just one task). We could probably do away with this term since we don't actually use it anywhere in the implementation.
LSS - stands for logical store shard. This is 1:1 with a Kafka Streams store at the task level, and therefore 1:1 with a Kafka partition. The expectation from RS3 is that each LSS has it's own WAL, which is the changelog topic-partition for the store. 
PS - Stands for physical store. This is a single store in RS3 that spans the whole RS3 cluster and stores data from one LS.
PSS - Stands for physical store shard. This is a single partition/shard of the PS that runs on one RS3 node as a slatedb instance. This is 1:1 with a TablePartition.